### PR TITLE
Mobilsdk 2202

### PR DESCRIPTION
**Pull request by lwburk**

### CODE REVIEW

File: InternationalizationSample/InternationalizationSample/Launch Screen.storyboard [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                                <rect key="frame" x="0.0" y="626.5" width="375" height="20.5"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <color key="textColor" red="0.0" green="0.0" blue="0.0" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                            <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="i18n Sample" textAlignment="center" lineBreakMode="middleTruncation" baselineAdjustment="alignBaselines" minimumFontSize="18" translatesAutoresizingMaskIntoConstraints="NO" id="GJd-Yh-RWb">
```
Comment (awangvan) [inline InternationalizationSample/InternationalizationSample/Launch Screen.storyboard]: i18n :smiley: 👍

File: InternationalizationSample/InternationalizationSample/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -6,46 +6,6 @@
...
-"FSCouldNotLoadSurveyMessage"="Couldn't load the survey at this time. There could be a problem with your connectivity.";
-
-"FSWebCloseButtonText"="Done";
-"FSWebLoadErrorTitle"="Page Load Fail";
-"FSWebLoadURLNotWhitelistedErrorMessage"="Couldn't load page. Only whitelisted domains are permitted.";
-"FSWebLoadErrorMessage"="Couldn't load page at this time. There could be a problem with your connectivity.";
-"FSWebLoadErrorButtonText"="OK";
+"FORESEE_Invite_InSession_DeclineButton"="No, thanks";
+"FORESEE_Invite_InSession_AcceptButton"="Yes, I'll help";
+"FORESEE_Invite_InSession_InvitePage_Title"="We want your feedback!";
```
Comment (awangvan) [inline InternationalizationSample/InternationalizationSample/en.lproj/Localizable.strings]: I wonder why overwriting the strings here in the sample app? Aren't they already handled in the SDK (and are same strings)?

File: InternationalizationSample/InternationalizationSample/FSViewController.m [hunk 3]
```diff
@@ -11,14 +11,6 @@
...
-
-#pragma mark - View lifecycle
-
-- (void)viewDidLoad {
-  [super viewDidLoad];
-}
-
 # pragma mark - Actions
 
 - (void)triggerInvite:(id)sender {
```
Comment (awangvan) [inline InternationalizationSample/InternationalizationSample/FSViewController.m]: I wonder if we should add a "Reset state" button, as the other samples? So the clients would know the "proper way" to check eligibilities.

File: InternationalizationSample/InternationalizationSample/en.lproj/Localizable.strings (hunk 2 above)
Comment (lwburk) [reply InternationalizationSample/InternationalizationSample/en.lproj/Localizable.strings]: It's just to show the user how they can override them, if they want to

File: InternationalizationSample/InternationalizationSample/Launch Screen.storyboard (hunk 1 above)
Comment (lwburk) [reply InternationalizationSample/InternationalizationSample/Launch Screen.storyboard]: :)

**Pull request merged**